docs/axes/radial/linear.md

```diff
@@ -110,3 +111,7 @@ The following options are used to configure the point labels that are shown on t
 | `fontSize` | `number` | `10` | font size in pixels.
 | `fontStyle` | `string` | `'normal'` | Font style to use when rendering point labels.
 | `lineHeight` | <code>number&#124;string</code> | `1.2` | Height of an individual line of text (see [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height)).
+
+## Internal data format
+
+Internnally linear radial scale uses numeric data
```

Comment:
typo: Internally

Reply:
Have to get a new keyboard, I'm getting a lot of extra `n`'s